recipes/doxygen/all/conanfile.py

```diff
@@ -27,6 +27,7 @@ def _build_subfolder(self):
 
     _minimum_compiler_version = {
         "gcc": 5,
+        "Visual Studio": 15,
```

Comment:
Are you sure about this?
See https://github.com/conan-io/conan-center-index/pull/5201#issuecomment-820667244

Only `1.8.18` failed to complete.
And I think the problem was not due to MSVC but because of the CI infrastructure.

I am unable to see the build logs/errors though.

Reply:
That was the only remaining comment on the PR, so I assumed this is what it was waiting for.

Reply:
I removed this commit.